### ISSUE TITLE
show recently updated web services pages

### ISSUE THREAD

**Issue by ezio416** [openplanet-nl/nadeoapi-docs#22]
On https://webservices.openplanet.dev/ I think it would be helpful to have a section that lists the pages that were most recently updated

**Comment by davidbmaier** [comment]
Hmm, what's the use case here? If you're just looking for a history of changes, the docs are open-source so you can just check the [commit history](https://github.com/openplanet-nl/nadeoapi-docs/commits/main/).

**Comment by ezio416** [comment]
My bad, forgot there was a separate repo for that xdd. Most notable recent example is the v2/mapRecords change. Is it possible to just move this issue to that repo?

**Comment by davidbmaier** [comment]
I'm not sure some kind of edit history would be easy to implement given that it's all just Git-managed files without any edit history metadata.
Maybe a better alternative would be something like a little news page for important information about the API and the docs (essentially like a changelog of the most important items with some more detail). It would probably need to be manually maintained, but it's better than only announcing stuff on Discord I guess.
Would that cover what you're looking for? If you just want to know recent changes (including typo fixes and minor changes) I feel like the commit history is already enough.

**Comment by ezio416** [comment]
Yeah I think a news page would be good, but only for important changes like the one I mentioned. Extra notes or small things are probably not worth including on there imo
